newsrevenuehub/rev-engine
# News revenue hub  sc 3591 set hub strip publishable key as environment

#### What's this PR do?
This PR adds all of the front-end env vars to the django app and to the Heroku account, to be set on the window via a django request on the index.html of the React app.

Note: I've updated the code to handle json data via django template safely, though it doesn't seem to be working as expected.  Probably some formatting thing I'm unaware of.

#### How should this be manually tested?
Start up python server and navigate to localhost:8000 check window object to see if .ENV object is present.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
Please review to ensure I've included all necessary config vars.  I wasn't able to find a Google Analytics ID, for example.
